compiler/p/codegen/FPTreeEvaluator.cpp

```diff
@@ -1460,9 +1460,15 @@ TR::Register *OMR::Power::TreeEvaluator::i2fEvaluator(TR::Node *node, TR::CodeGe
    TR::Register *tempReg;
    TR::Register *trgReg;
 
+<<<<<<< HEAD
    if (((cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P7) &&
        (node->getOpCodeValue() == TR::iu2f && (child->getOpCodeValue() == TR::iuload || child->getOpCodeValue() == TR::iuloadi))) ||
        (cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P6) &&
+=======
```

Comment:
I'm not sure these hack markers should be here...

Reply:
Sorry for the rebasing error. I have fixed this problem in the new commit.